packages/ipfs-unixfs-exporter/src/resolvers/unixfs-v1/content/hamt-sharded-directory.ts

```diff
@@ -20,11 +22,28 @@ const hamtShardedDirectoryContent: UnixfsV1Resolver = (cid, node, unixfs, path,
 async function * listDirectory (node: PBNode, path: string, resolve: Resolve, depth: number, blockstore: ReadableStorage, options: ExporterOptions): UnixfsV1DirectoryContent {
   const links = node.Links
 
+  if (node.Data == null) {
+    throw errCode(new Error('no data in PBNode'), 'ERR_NOT_UNIXFS')
+  }
+
+  let dir: UnixFS
+  try {
+    dir = UnixFS.unmarshal(node.Data)
+  } catch (err: any) {
+    throw errCode(err, 'ERR_NOT_UNIXFS')
+  }
+
+  if (!dir.fanout) {
```

Comment:
To fix the linting error:

```suggestion
  if (dir.fanout == null) {
```